### PR TITLE
Updates to `/account` page

### LINKED ISSUE #268: User account landing page updates
- [x] update title to “Learning”
- [x] update text at the top
- [x] update header text size (changed to h1 and h2 for the breakpoints)
- [x] remove “groups”
- [x] remove “badges”
- [x] remove user title (eg. admin)
- [x] if user firstName and lastName are empty values, display 'Your Profile' 
- [x] logout - user should stay on current page after logged out (if user is on the `/account` page, redirect to login page)
- [x] make text under header be 8 columns wide instead of full width

Note: the content of this page is a work in progress in figma, but it should be the course material

Design file [here](https://www.figma.com/file/SOnOgCGQIWx4oAlSFhIsIX/1972-User-account?node-id=1114%3A7615)

### PR DESCRIPTION
**Pull request by techtolentino**
Fixes #268 

Notes for the reviewer:
- pull down branch to local machine
- comment out [this line](https://github.com/Qiskit/platypus/pull/341/files#diff-226c1869e16c3e24e492b3483e64e2c58cca766e43b81632c01fb056ff8f666dR92)
- build and start app
- navigate to `localhost:8080/account`

### CODE REVIEW

File: frontend/vue/components/UserAccount/UserAccountLayout.vue [hunk 1]
```diff
@@ -59,41 +52,35 @@
...
-          displayName: this.$translate('Badges'),
-          hash: '#Badges'
-        },
         {
           displayName: this.$translate('Privacy'),
           hash: '#Privacy'
         }
-      ]
+      ],
+      logoutRoute: '/logout'
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/UserAccountLayout.vue:83]: @vabarbosa - I'll need some guidance here, to figure out  how we tie this in w/ Mathigon's `/logout`. I don't think this is the complete piece, and I'm guessing I'll need to add something to `app.ts`?

Comment (Tansito) [reply frontend/vue/components/UserAccount/UserAccountLayout.vue:83]: In mathigon we have a [`logout` end-point](https://github.com/Qiskit/platypus/pull/174/files#diff-6eef9123bed920a90b44258c0f446f14d024a1a2f14b80517f0daf2c6b29ac42R149) that removes the session and redirects to `back` (I don't know exactly what is that). So initially you don't need to add anything in `app.ts` but I can take a look tomorrow.

Comment (vabarbosa) [reply frontend/vue/components/UserAccount/UserAccountLayout.vue:83]: `redirect('back')` should send the user back to the referer of `/` (if no referer):
https://expressjs.com/en/4x/api.html#res.redirect

Comment (vabarbosa) [reply frontend/vue/components/UserAccount/UserAccountLayout.vue:83]: FYI, i just tested locally (using google) and logout worked as expected

**Review by vabarbosa — APPROVED** [review]
@techtolentino thank you!

**Pull request merged**